exercises/concept/library-fees/.docs/instructions.md

```diff
@@ -0,0 +1,61 @@
+Your librarian friend has asked you to extend her library software to automatically calculate late fees.
+Her current system stores the exact date and time of a book checkout as an ISO8601 datetime string.
+She runs a local library in a small town in ..., which uses the GMT timezone (UTC +0), and doesn't need to worry about other timezones.
```

Comment:
@iHiD I'm not sure which country to choose here. It doesn't matter for the exercise at all, but I didn't want to be Europe-centric yet again... Due to my white privilege and not paying attention in geography class, I don't know much about Africa. I would want to avoid accidentally naming a country with gross human right violations, and where a woman running a library on her own wouldn't even be possible. Here are the GMT African countries:
- Mali
- Morocco
- Western Sahara
- Mauritania
- The Gambia
- Senegal
- Guinea-Bissau
- Guinea
- Sierra Leone
- Liberia
- Burkina Faso
- Côte d'Ivoire
- Ghana

Any recommendations? Or should I just stick to what I know and choose Northern Ireland or something?

Reply:
Thanks for thinking/caring! :)

Mali and Ghana are solid choices. I believe Senegal is making good strides forward with Human Rights. I don't know about the others.